src/txmempool.h

```diff
@@ -865,9 +856,25 @@ class CTxMemPool
         // map from the m_to_add index to the ancestors for the transaction
         std::map<CTxMemPool::txiter, CTxMemPool::setEntries, CompareIteratorByHash> m_ancestors;
         CTxMemPool::setEntries m_to_remove;
+
+        friend class CTxMemPool;
     };
 
     std::unique_ptr<ChangeSet> GetChangeSet() EXCLUSIVE_LOCKS_REQUIRED(cs) { return std::make_unique<ChangeSet>(this); }
+
+    friend class CTxMemPool::ChangeSet;
+
+private:
+    // addNewTransaction must updated state for all ancestors of a given transaction,
```

Comment:
while we're moving the comment, can we fix the grammar: "must updated"?

Reply:
Fixed.